tests/topotests/ldp-vpls-topo1/r1/show_ldp_binding.ref

```diff
@@ -20,31 +20,27 @@
       "addressFamily":"ipv4",
       "prefix":"2.2.2.2/32",
       "neighborId":"2.2.2.2",
-      "localLabel":"17",
       "remoteLabel":"imp-null",
       "inUse":1
     },
     {
       "addressFamily":"ipv4",
       "prefix":"2.2.2.2/32",
       "neighborId":"3.3.3.3",
-      "localLabel":"17",
```

Comment:
I believe this topotest can still fail with these changes since the `remoteLabel` of one router is the `localLabel` of another one (so `remoteLabel` is also unpredictable). I think we need to either a) stop checking the `remoteLabel` as well or b) only check if the local/remote labels are equal to `imp-null` or not.

Reply:
@mwinter-osr can you please respond to this or close the PR? This review comment is getting close to 2 months old 🙂